torchvision/prototype/datasets/_builtin/__init__.py

```diff
@@ -1,3 +1,4 @@
+from . import imagenet
```

Comment:
Since we register through the decorator now, we don't need to explicitly import anything anymore. We just need to make sure the decorator is executed.

Reply:
We also don't even need to import anything here, right? Since the functions would become public we just have to expose them in `datasets.__init__.py`

Reply:
Should the functions be exposed at `torchvision.prototype.datasets`?